charts/linkerd-crds/templates/policy/server.yaml

```diff
@@ -148,6 +148,11 @@ spec:
                 - required: [podSelector]
                 - required: [externalWorkloadSelector]
               properties:
+                accessPolicy:
```

Comment:
Since we are adding a new field to the resource, should we also bump the resource version?

Reply:
I was relying on this being BC to not bump the version, but good call, if someone wants to use this field the only way to guarantee it's supported is through the version.